src/qml/DockPage.qml

```diff
@@ -109,6 +109,32 @@ ItemPage {
                 }
             }
 
+            Item {
+                Layout.fillHeight: true
+                Layout.fillWidth: true
+            }
+            
+            RowLayout {
```

Comment:
another nitpick: could you add a separator here? you can see these between Position and Size

Reply:
doing it rn